packages/react/src/components/asorted/Text/styles.ts

```diff
@@ -64,59 +64,70 @@ export const textVariantStyle: Record<
     fontFamily: string;
     lineHeight?: string;
     fontWeight?: number;
+    "text-transform"?: string;
   }
 > = {
   h1: {
     fontFamily: "Alpha",
     fontWeight: 500,
+    "text-transform": "uppercase",
   },
   h2: {
     fontFamily: "Alpha",
     fontWeight: 500,
+    "text-transform": "uppercase",
   },
   h3: {
     fontFamily: "Alpha",
     fontWeight: 500,
+    "text-transform": "uppercase",
   },
   h4: {
     fontFamily: "Inter",
```

Comment:
⚠️ Not directly linked to this PR, but the font family is wrong here it should be `Alpha`

Reply:
Good catch thanks, i'm gonna fix it right away & figure out the lineheight issue